stubs/pycocotools/METADATA.toml

```diff
@@ -1,2 +1,5 @@
 version = "2.0.*"
 upstream_repository = "https://github.com/ppwwyyxx/cocoapi"
+requires = ["numpy"]
+# numpy>=2.0.0 requires Python >=3.9
+requires_python = ">=3.9"
```

Comment:
pycocotools doesn't require `numpy>=2.0.0`, right ? I don't understand that comment.